pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.3
+title: Release notes for 2.0.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.3, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.3 was released on 19 March 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/2.0.3/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.3/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release Summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.3 includes the following component versions:
+
+- Apache&reg; Mesos&reg; 1.8.2-dev
+- OpenSSL 1.1.1d	
+
+### DC/OS Fixed and Improved Issues
```

Comment:
We're missing an entry for https://jira.d2iq.com/browse/COPS-4995.   The Mesos fix is here: https://github.com/apache/mesos/commit/a2ca451aab4625e126b9e7b470eb9f7c232dd746

Reply:
added